Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -11,7 +11,12 @@ class TestAppUITests: XCTestCase {
     }
 
     func testIsLoadedReturnsTrueWhenScreenIsLoaded() throws {
-        let screen = try HelloWorldScreen()
+        let screen = try HelloWorldScreen(app: app)
+        XCTAssertTrue(screen.isLoaded)
+    }
+
+    func testIsLoadedReturnsTrueWhenScreenWithMultipleElementsIsLoaded() throws {
+        let screen = try MultipleElementsScreen(app: app)
         XCTAssertTrue(screen.isLoaded)
     }
```

Comment:
Similarly to the test a few lines above, this simply creates a screen that matches the one loaded in the app and checks that `isLoaded` returns true.

The only difference is that `MultipleElementScreen` uses two expected element getter closures.